app/components/widgets/forms/location-input.js

```diff
@@ -24,7 +25,10 @@ export default Component.extend({
 
   actions: {
     showAddressView(show = true) {
-      this.set('addressViewIsShown', show);
+      let showAddress = this.get('showButton');
```

Comment:
Why is this required ? without the button this action will anyway not be triggered right ?